components/inquiry_questionnaire/map/decisions.coffee

```diff
@@ -4,6 +4,10 @@ hasSeen = (steps...) -> ({ logger }) -> logger.hasLogged steps...
 hasSeenThisSession = (steps...) -> ({ logger }) -> logger.hasLoggedThisSession steps...
 
 decisions =
+  is_auction: ({ artwork }) ->
+    artwork.related()
+      .partner.get('type') is "Auction"
```

Comment:
Cool cool! Feel free to merge when you are ready. Thanks a lot for the help here! 🙇